examples/date-time/date_time.bal

```diff
@@ -11,63 +11,63 @@ public function main() {
     // time, and timezone information.
     time:Time timeCreated = time:createTime(2017, 3, 28, 23, 42, 45, 554,
                                             "America/Panama");
-    io:println("Created Time: " + timeCreated.toString());
+    io:println("Created Time: " + time:toString(timeCreated));
     // This retrieves the time for a given string representation
     // based on the specified string format.
     time:Time t1 = time:parse("2017-06-26T09:46:22.444-0500",
                               "yyyy-MM-dd'T'HH:mm:ss.SSSZ");
-    io:println("Parsed Time: " + t1.toString());
+    io:println("Parsed Time: " + time:toString(t1));
     // You can retrieve the string representation of the time via the `toString()` function or the `format()` function.
     // This fetches the ISO 8601 formatted string of a given time.
-    string standardTimeString = time.toString();
+    string standardTimeString = time:toString(time);
```

Comment:
Shall we rename this `time` variable to a different name? May be `currentTime`, as it increases the readability(Otherwise having time as the package name and variable names seems bit misleading) We can address this in a different PR.